help/en/package/apps/jmrit/log/Log4JTreePane.shtml

```diff
@@ -13,13 +13,13 @@
     <div id="mainContent" class="no-sidebar">
       <h1>JMRI: Display / Edit Log Categories</h1>
       
-      <p>This tool can be opened via Debug -&gt; Display / Edit Log Categories</p>
+      <p>This tool can be opened via Debug &rarr; Display / Edit Log Categories</p>
```

Comment:
The convention for menu paths is to use the `&rArr;` entity (=>) enclosed in **strong** tags.

Reply:
Happy to make that change in a moment. I did not notice that convention in the Development instructions.